save-plugins/warn-plugin/src/commonMain/kotlin/org/cqfn/save/plugin/warn/WarnPluginConfig.kt

```diff
@@ -101,14 +134,14 @@ data class WarnPluginConfig(
     companion object {
         /**
          * Default regex for expected warnings in test resources, e.g.
-         * `// ;warn:2:4: Class name in incorrect case`
+         * `// ;warn:ClassNameTest.java:2:4: Class name in incorrect case`
```

Comment:
But you haven't change the pattern itself, only comment. Also, I think for input pattern it's no loner necessary